src/k8s/clone.js

```diff
@@ -136,12 +140,21 @@ const addPvcClones = (vm, newName, persistentVolumeClaims) => {
     });
 };
 
-export const addTemplateClone = (vm, pvcName, pvcNamespace, accessModes, size, storageClassName, vmName) => {
-  const template = getPvcCloneTemplate(pvcName, pvcNamespace, accessModes, size, storageClassName, vmName);
+export const addTemplateClone = (
```

Comment:
ugh, how did we get to this? These parameters are quite unmaintainable..

Reply:
right, we need to refactor. In this particular case, I'm just adding `volumeMode`